packages/cxl-ui/src/components/cxl-course-card.js

```diff
@@ -27,9 +27,9 @@ export class CXLCourseCardElement extends CXLBaseCardElement {
   }
 
   _renderHeaderName() {
-    return html`<div class="name" title=${this.name}>
+    return html`<h2 class="name" title=${this.name}>
```

Comment:
This heading seems to be too big

![Screenshot 2024-01-05 at 09 25 17](https://github.com/conversionxl/aybolit/assets/5165721/4f8a01e7-1380-4cbd-8143-424f34db5d8c)